plugins/marathon/marathon.go

```diff
@@ -163,8 +163,8 @@ func (r *marathonRunner) Run(cmd *command.Cmd, args []string) command.CmdErr {
 	// Initialize the collector and start updating
 	collector = marathonCollector{
 		marathonCollectorSettings: r.marathonCollectorSettings,
-		client: mclient,
-		filter: filter,
+		client:                    mclient,
+		filter:                    filter,
```

Comment:
what version of go are you on? we run `go fmt` as part of our CI, and this passes as-is.